lib/tlsca/ca.go

```diff
@@ -1345,6 +1346,7 @@ func (ca *CertAuthority) GenerateCertificate(req CertificateRequest) ([]byte, er
 		"dns_names", req.DNSNames,
 		"key_usage", req.KeyUsage,
 		"common_name", req.Subject.CommonName,
+		"issuer_skid", base32.HexEncoding.EncodeToString(ca.Cert.SubjectKeyId),
```

Comment:
I wonder if (in the case of HSM) the audit event should include the path/ID of the signing key in the HSM, which would be more easily identifiable than just the SKID.

@nklaassen is this info readily available?

Reply:
well it's available, that's what we actually store in the backend CA object, each keystore backend implementation in lib/auth/keystore current parses it's own key ID format it's just not exported right now

Reply:
I haven't found a good way to expose it, I'll leave it as it is for now if that's OK